ctrd/utils.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/pkg/errors"
 )
 
+// pushTracker records push progress detail
+var pushTracker = docker.NewInMemoryTracker()
```

Comment:
I am not sure that it is good to use global variables. There is one case:

Someone has pushed image A with desc A1. If other one tries to push image B which also contains desc A1, the status might be `done`. But it might be under committing.

Reply:
It remind me, looks like a parallel caused bug, we can also do a fix for containerd

Reply:
please remove this line since you uses temp var.

Reply:
containerd ctr is command , not a server implement. it doesn't have the issue.